int32 obj_num
int32 obj_class
float64 x
float64 y
float64 x_d
float64 y_d
float64 x_dd
float64 y_dd
float64 yaw
float64 yaw_rate
float64 w
float64 h